core/src/main/scala/scala/collection/parallel/mutable/ParHashMap.scala

```diff
@@ -196,7 +196,7 @@ extends scala.collection.parallel.BucketCombiner[(K, V), ParHashMap[K, V], Defau
   } else {
     // construct a normal table and fill it sequentially
     // TODO parallelize by keeping separate sizemaps and merging them
-    object table extends HashTable[K, DefaultEntry[K, V], DefaultEntry[K, V]] with WithContents[K, DefaultEntry[K, V], DefaultEntry[K, V]] {
+    object newTable extends HashTable[K, DefaultEntry[K, V], DefaultEntry[K, V]] with WithContents[K, DefaultEntry[K, V], DefaultEntry[K, V]] {
```

Comment:
I called it `normal`...

Also, I see that I handled `seedvalue` differently at L232 below:
```
-    seedvalue = _seedvalue
+    this.seedvalue = _seedvalue
```
I don't remember what all the conflicts were, but when the new rule kicks in, the inherited member (which was picked in scala 2) can still be selected.

Reply:
good call. I included this in https://github.com/scala/scala-parallel-collections/pull/118